pkg/notifier/WebhookNotificationService.go

```diff
@@ -0,0 +1,237 @@
+package notifier
+
+import (
+	"fmt"
+	"github.com/devtron-labs/devtron/internal/sql/repository"
+	"github.com/devtron-labs/devtron/internal/util"
+	"github.com/devtron-labs/devtron/pkg/sql"
+	"github.com/devtron-labs/devtron/pkg/team"
+	repository2 "github.com/devtron-labs/devtron/pkg/user/repository"
+	util2 "github.com/devtron-labs/devtron/util/event"
+	"github.com/go-pg/pg"
+	"go.uber.org/zap"
+	"time"
+)
+
+const WEBHOOK_CONFIG_TYPE = "webhook"
+
+type WebhookAttribute string
+
+const (
```

Comment:
add comments, over these fields will be configurable in future

Reply:
done